src/commands/dev/lib/dev-builder.ts

```diff
@@ -228,7 +238,9 @@ export async function executeBuild(
         }
         function onExit(code: number | null, signal: string | null) {
           cleanup();
-          const err = new Error(`Builder exited with ${signal || code} before sending build result`);
+          const err = new Error(
+            `Builder exited with ${signal || code} before sending build result`
```

Comment:
Why is the formatting changing here? Is this your IDE or is this the commit hook?

Reply:
It's prettier

Reply:
It's `prettier` 🤷‍♂